docs/shared/packages/cypress/cypress-plugin.md

```diff
@@ -81,18 +81,28 @@ To enable e2e task splitting, make sure there is a `ciWebServerCommand` property
 import { defineConfig } from 'cypress';
 import { nxE2EPreset } from '@nx/cypress/plugins/cypress-preset';
 
+const config = nxE2EPreset(__filename, {
+  cypressDir: 'src',
+  bundler: 'vite',
+  webServerCommands: {
+    default: 'nx run my-project:serve',
+    production: 'nx run my-project:preview',
+  },
+  ciWebServerCommand: 'nx run my-project:serve-static',
+});
+
 export default defineConfig({
   e2e: {
-    ...nxE2EPreset(__filename, {
-      cypressDir: 'src',
-      bundler: 'vite',
-      webServerCommands: {
-        default: 'nx run my-project:serve',
-        production: 'nx run my-project:preview',
-      },
-      ciWebServerCommand: 'nx run my-project:serve-static',
-    }),
+    ...config,
     baseUrl: 'http://localhost:4200',
+
+    // If you set `setupNodeEvents`, make sure to invoke the function from `nxE2EPreset`
+    // See: https://docs.cypress.io/guides/references/configuration#setupNodeEvents
+    async setupNodeEvents(on, config) {
+      await config.setupNodeEvents(on, config);
+
+      // Register your listeners here
+    },
```

Comment:
I don't think this is relevant for this particular section of the docs. Here we're showing an example of splitting e2e tasks, and we're particularly focused on adding the `ciWebServerCommand`. The sentences leading to this code snippet describes that:

```
To enable e2e task splitting, make sure there is a `ciWebServerCommand` property set in your `cypress.config.ts` file. It will look something like this:
```

And we explicitly highlight the line where we set that option.

I think this could go into a recipe and be linked from the plugin page. If we want to keep it on the plugin page, it should probably be a separate section/example focusing on `setupNodeEvents`.